### PR TITLE
my html task

### PR DESCRIPTION
**Pull request by idayatOlaide**
This my portfolio html task

### CODE REVIEW

**Comment by Temitoyin** [comment]
Hi Idayat, I think your portfolio is well arranged and your use of tags is great. I noticed the main tag is missing an opening tag and some typographical errors in your bio section. Other than that, good work.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            </a>
+        </div>
+    </section>
+
+    <!-- footer -->
+    <footer id="main-footer" class="bg-dark text-center py-1">
+        <div class="base">
+            <P>Copyright &copy; 2019, IdayatPFolio, All Rights Reserved</P>
+        </div>
+    </footer>
```
Comment (Temitoyin) [inline index.html:110]: This is the main closing tag, but I can't see the main opening tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                        March 2020 - Till Present</p>
+                </div>
+            </div>
+            <section id="content">
+                <div class="content-img"></div>
+                <div class="content-text bg-dark p-1">
+                    <h1 class="m-heading text-center"><span class="text-primary">About</span> Me
+                    </h1>
+                    <P class="sub">Professional Path</P>
+                    <p class="bio">
```
Comment (Temitoyin) [inline index.html]: The typographical error is almost negligible It's the "six months" and last "i" should be in capital letter

**Review by Temitoyin — COMMENTED** [review]
I really like the use of your class names and Ids they are very descriptive

File: index.html (hunk 2 above)
Comment (idayatOlaide) [reply index.html]: I have made changes to it and Also commit the changes @Temitoyin thanks

File: index.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Idayat Portfolio</title>
+    <!--<link href="./CSS folder/index.css" rel="stylesheet">-->
```
Comment (chiamakaikeanyi) [inline index.html]: This is not needed at this point

File: index.html [hunk 4]
```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (chiamakaikeanyi) [inline index.html:5]: Add other meta tags -  title, keywords, description etc

File: index.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+                    <p>FrontEnd developer<br>
+                        March 2020 - Till Present</p>
+                </div>
+            </div>
+            <section id="content">
+                <div class="content-img"></div>
+                <div class="content-text bg-dark p-1">
+                    <h1 class="m-heading text-center"><span class="text-primary">About</span> Me
+                    </h1>
+                    <P class="sub">Professional Path</P>
```
Comment (chiamakaikeanyi) [inline index.html]: Use lower case here

File: index.html [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+                    <p class="bio">
+                        I am Idayat Abdulkareem,software developer. A graduate of Unilorin<br>
+                        currently working with Konga as a FrontEnd Developer.<br>
+                        I started coding six months ago and I enjoy coding.
+                    </p>
+                </div>
+        </div>
```
Comment (chiamakaikeanyi) [inline index.html]: Use proper indentation. Consider using prettier and format on save if you use VSCode

File: index.html [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+            </div>
+            <div class="contact__address">
+                <i class="fas fa-location-arrow"></i> Address<br>
+                <div class="contact-details">33 Airport Road, Off Ajao-Estate. Lagos.</div>
+            </div>
+        </div>
+    </section>
+    <section id="icon" class="py-1">
+        <div class="social-icon" class="py-1">
+            <a href="https://twitter.com" target="blank">
```
Comment (chiamakaikeanyi) [inline index.html]: Note that when you use target="_blank", always add rel="noopener" or rel="noreferrer". Visit https://developers.google.com/web/tools/lighthouse/audits/noopener for more information.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+            <a href="https://codepen.io" target="blank">
+                <span class="text-primary"> <i class="fab fa-codepen fa-2x"></i></span>
+            </a>
+        </div>
+    </section>
+
+    <!-- footer -->
+    <footer id="main-footer" class="bg-dark text-center py-1">
+        <div class="base">
+            <P>Copyright &copy; 2019, IdayatPFolio, All Rights Reserved</P>
```
Comment (chiamakaikeanyi) [inline index.html]: This should be 2020

File: index.html [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+        </div>
+    </footer>
+    <script src="https://kit.fontawesome.com/4b5f45f1a6.js" crossorigin="anonymous"></script>
```
Comment (chiamakaikeanyi) [inline index.html]: `<script>` is not meant to be seen at this point

**Pull request merged**